recipes/corretto.rb

```diff
@@ -1,46 +1,10 @@
-# Cookbook:: java
-# Recipe:: corretto
-# This recipe installs and configures Amazon's Corretto package
-# https://aws.amazon.com/corretto/
 
-include_recipe 'java::notify'
+Chef::Log.fatal("
 
-unless node.recipe?('java::default')
-  Chef::Log.warn('Using java::default instead is recommended.')
+Corretto recipe is now deprecated
+Using the corrett_install resource is now recommended
+See: documentation/resources/install_corretto.md for help
```

Comment:
This should be a full link back to the repo so someone can take action when they see this in their log